motr/client_init.c

```diff
@@ -1600,9 +1600,11 @@ int m0_client_init(struct m0_client **m0c_p,
 
 	if (conf->mc_is_addb_init) {
 		char buf[64];
+		/** Client addb record file size set to 1G" */
+		m0_bcount_t size = 1ULL >> 30;
```

Comment:
I think this is a typo. It should be:

m0_bcount_t size = 1ULL << 30;  /* 1 Giga */

Reply:
Corrected. e3b8f9e